### DISCUSSION TITLE
How to turn on or off SSL verification with aiohttp.request() ?

### DISCUSSION THREAD

**Discussion by jsathish** [aio-libs/aiohttp#5786] [Q&A] [answered]
Thanks to all those people who have spent their valuable time and contributed to the aiohttp.

I am new to using the aiohttp library. When I went through the docs, noticed `aiohttp.request()` function takes up an argument "ssl". But when I tried to use the same option in code, I am getting a TypeError invalid argument.

```python
class aiohttp.ClientSession(*, connector=None, loop=None, cookies=None, headers=None, skip_auto_headers=None, auth=None, json_serialize=json.dumps, version=aiohttp.HttpVersion11, cookie_jar=None, read_timeout=None, conn_timeout=None, timeout=sentinel, raise_for_status=False, connector_owner=True, auto_decompress=True, read_bufsize=2 ** 16, requote_redirect_url=False, trust_env=False, trace_configs=None)¶
```

May I know what parameters I have to use to turn on or off the SSL verification.
I am working on python 3.6.8 and aiohttp 3.7.4.post0

**Comment by webknjaz** [comment]
Show us the code.

**Comment by Dreamsorcerer** [comment, marked as answer]
> May I know what parameters I have to use to turn on or off the SSL verification.

Well, the documentation you referred to describes the valid arguments as:
> SSL validation mode. None for default SSL check (ssl.create_default_context() is used), False for skip SSL certificate validation, aiohttp.Fingerprint for fingerprint validation, ssl.SSLContext for custom SSL certificate validation.
https://docs.aiohttp.org/en/stable/client_reference.html#client-session

If you're getting a `TypeError` while passing in one of these, then feel free to file a bug report, and include the code you are trying.
